shiny/Data-Ingest/server_files/ingest_workflow_svr.R

```diff
@@ -129,18 +129,25 @@ updateSelectizeInput(session, "MimetypeName", choices = mimetypes, server = TRUE
 
 observeEvent(input$createFormatRecord, {
   ## MimetypeID
-  FormatRecordList$mimetypeName <- input$MimetypeName
-  FormatRecordList$mimetypeID <- mimetype_sub %>% dplyr::filter(type_string %in% FormatRecordList$mimetypeName) %>% pull(id)
+  FormatRecordList$mimetypeName <<- input$MimetypeName
+  FormatRecordList$mimetypeID <<- mimetype_sub %>% dplyr::filter(type_string %in% FormatRecordList$mimetypeName) %>% pull(id)
   
   ## Everything else
-  FormatRecordList$NewMimeType <- input$NewMimeType
-  FormatRecordList$NewFormatName <- input$NewFormatName
-  FormatRecordList$HeaderBoolean <- input$HeaderBoolean
-  FormatRecordList$SkipLines <- input$SkipLines #This should appear only if header = TRUE
-  FormatRecordList$FormatNotes <- input$FormatNotes
+  FormatRecordList$NewMimeType <<- input$NewMimeType
+  FormatRecordList$NewFormatName <<- input$NewFormatName
+  FormatRecordList$HeaderBoolean <<- ifelse((input$HeaderBoolean == "Yes"), TRUE, FALSE)
+  FormatRecordList$SkipLines <<- input$SkipLines #This should appear only if header = TRUE
+  FormatRecordList$FormatNotes <<- input$FormatNotes
   
   output$FormatRecordOut <- renderPrint({print(FormatRecordList)})
   
+  cmd <- paste0("INSERT INTO formats ",
```

Comment:
* This needs to be it's own function in the DB package, not here in the shiny app.
* This function needs to return the new Format ID.
* I'm fairly confident you don't need to include the NOW() bits -- the existence of that in other functions is a legacy that I thought had been cleaned up, but evidently not as you're emulating it
* This function also needs to take an (optional) formats-variables table and insert all those relationships.

Reply:
Got it. I can drop the NOW() bits, but there are columns for date created and date updated in the formats table. Just want to confirm that you want to lose that functionality? 

Would it be easier/ more efficient to create a separate format-variables table query since I'd be writing to two separate tables? Just double checking before I get started. Thanks!

Reply:
NOW: I'm 99% sure that those will be updated by postGRES automatically if you don't manually set them. I'd drop and then confirm that they're being set.

Creating Formats and Formats-Variables separately is exactly the pain in arse that we currently have to deal with in the BETY webpage and specifically what we're trying to avoid.  Exactly analogous to why we create dbfiles and inputs at the same time.

Reply:
NOW: sounds good. 

F-V: Ok got it. Just to clarify, by the same time, you mean in the same function? My poin

Reply:
Would you like me to resolve the `mimetype_id` in this new db.files function? I am currently resolving the ID within the shiny app. Thanks!

Reply:
Yes, in the same function

mimetype_id would need to be an input to the function

Reply:
OK! This revision should have all the changes you asked for!